Dockerfile

```diff
@@ -21,7 +21,7 @@ RUN apt-get update -y && \
 	apt-get install -y \
 		git \
 		unzip && \
-	curl -Ls -o /usr/bin/k8s-claimer https://storage.googleapis.com/k8s-claimer/git-e4dcc16/k8s-claimer-git-e4dcc16-linux-amd64 && \
+	curl -Ls -o /usr/bin/k8s-claimer https://k8sclaimercli.blob.core.windows.net/cli/k8s-claimer-latest-linux-amd64 && \
```

Comment:
I missed this before, but realized from what @sgoings said that I think this should probably be versioned @jchauncey